Site-Reliability-Engineering/L3-Senior-Site-Reliability-Engineer.md

```diff
@@ -0,0 +1,82 @@
+# L3: Senior Site Reliability Engineer
+
+> _I am a highly sought-after member of an Octopus team with a proven record of independently identify and deliver solutions that ensure the resilience and reliability across a broad product capability or technical system._
+
+- **Planning horizon**: Current and next cycle
+- **Impact radius**: Peers (2-5)
+- **Evaluation**: Manager
+- **Responsibility and direction needed**: Takes vertical slices of larger products from idea to production.
+
+## 🦉 Domain expertise
+
+- I am advanced in my team's domain.
+- I can confidently step outside my own comfort zone and adapt quickly to new technologies.
+- I am able to codify conventions (where appropriate) into my team\'s systems
+- I make appropriate scaling, reliability, and maintenance trade-offs as they occur in practice.
+- I identify, qualify and reduce technical debt as appropriate to my team's priorities.
+
+<details>
+<summary>Examples</summary>
+
+- I set up or maintained an automated integration and delivery pipeline.
```

Comment:
suggest add "improved" to setup or maintain.

Reply:
Done